android/libraries/rib-android/src/main/kotlin/com/uber/rib/core/RibActivity.kt

```diff
@@ -157,10 +158,13 @@ abstract class RibActivity :
     callbacksFlow.tryEmit(createTrimMemoryEvent(level))
   }
 
+  @RequiresApi(api = Build.VERSION_CODES.O)
+  @CallSuper
   override fun onPictureInPictureModeChanged(
     isInPictureInPictureMode: Boolean,
     newConfig: Configuration,
   ) {
+    super.onPictureInPictureModeChanged(isInPictureInPictureMode, newConfig)
```

Comment:
@jbarr21 I think that's a new requirement because of the `androidx.activity` bump (I believe @CallSuper was added); from `super` source code I don't think there's any issues in calling super. 

Tagging you just in case you know something that I don't.